doc/json-schemas/build-info.schema.json

```diff
@@ -0,0 +1,83 @@
+{
```

Comment:
The schema should also link to the code and the other way round.

Reply:
Should be done